syncback/Tiltfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: Python -*-
+
+krsync_path = os.path.join(os.getcwd(), 'krsync.sh')
+
+
+def syncback(name, k8s_object, paths, src_dir, target_dir='.', namespace=''):
+    # name (str) = resource name
+    # k8s_object (str) = thing to exec on (pod/foo, deployment/bar, etc)
+    # paths (List[str]) = paths ON CONTAINER to sync. May be files or dirs. May be absolute, or relative to src_dir
```

Comment:
What path would you use if you wanted everything under the root to be synced back if it's changed in-container, and does that work OK?

Reply:
good call, this is a use case i hadn't considered. EXTREMELY counterintuitively, the path `''` will get everything in `src_dir`. Let me change it to make `paths` optional, and passing NO paths defaults to pulling everything from `src_dir`.

Reply:
☝️ does that sound like reasonable behavior?

Reply:
Yeah, that sounds reasonable.